### ISSUE TITLE
Using BungeeForge Causes Proxy Registered Commands to not Register

### ISSUE THREAD

**Issue by Aiidoneus** [caunt/BungeeForge#4]
Hello,
I was testing this solution on my 1.19 servers (FTB Skies) and any Velocity plugins that register commands like Huskchat or EnhancedVelocity for example do not auto-complete. I switched from Proxy Compatible Forge to this one and it seems to not register it on the client for some reason. The commands still work but they say they don't. 

Any ideas?

**Comment by caunt** [comment]
<img width="239" alt="image" src="https://github.com/caunt/BungeeForge/assets/5324218/0c089dbe-594e-4a4f-9a2d-1a47c229170e">

Just tested EnhancedVelocity + FTB Skies setup with "none" forwarding mode (without any PCF nor BungeeForge)
Command is still unknown to client, that means the problem is within Velocity or Ambassador itself. Did it work with PCF? If so, probably the PCF has implemented command-forwarding, not only player-related information forwarding

**Comment by caunt** [comment]
Yes, PCF itself has implemented command forwarding using Ambassador to scrape commands from Velocity. I could potentially use this opportunity to register commands on the client, but I don't guarantee that it will work later in newer versions

**Comment by Aiidoneus** [comment]
Yes. Looks like you answered most of your questions.

**Comment by caunt** [comment]
I realized that BungeeForge is compatible with PCF to be installed on the same server. PCF author also mentioned that:

<img width="556" alt="image" src="https://github.com/caunt/BungeeForge/assets/5324218/600f83d1-55ca-43d9-9b16-19ff4341fe28">


Simply install both BungeeForge and PCF. BungeeForge will forward your player information, PCF will forward Velocity commands information. Tested it with All The Mods 9, command forwarding works fine 🙂
